recipe/meta.yaml

```diff
@@ -26,6 +26,8 @@ requirements:
     - resolve-robotics-uri-py
     - python >=3.8
```

Comment:
Sorry for not noticing before, there is also this missing bump:

```suggestion
    - python >=3.10
```

Reply:
Done in https://github.com/conda-forge/rod-feedstock/pull/8/commits/7961a3bd9bc30f11711d43f34efd48f98dd55e12